src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -96,34 +92,47 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of using escapes with trail comment:
+ * Example of using escapes for non-printable, control characters:
  * </p>
  * <pre>
- * String unitAbbrev = "&#92;u03bcs"; // Greek letter mu, "s"
- * String textBlockUnitAbbrev = """
- *          &#92;u03bcs"""; // Greek letter mu, "s"
+ * String unitAbbrev = "μs"; // No violation, a normal String
```

Comment:
`// OK, a normal String `

please apply to all other lines like this.

Reply:
done

Reply:
I still see "no violation" on some lines